portfolio/src/main/webapp/script.js

```diff
@@ -128,8 +128,10 @@ function showSlides(n) {
  * Fetch data from the server.
  */
 function getServerData() {
-  fetch('/data').then(response => response.text()).then((myJSON) => {
-    document.getElementById('data-container').innerText = myJSON;
-  });
+  fetch('/data')
+    .then(response => response.text())
```

Comment:
indent this line by 4 spaces instead of 2 because it's a continuation of the previous line, not a new statement.